Jenkinsfile

```diff
@@ -6,19 +6,24 @@ def jobParams = [
   booleanParam(defaultValue: false, description: 'Upload builds to nexus(master,develop and staging branches upload always)', name: 'upload_to_nexus'),
 ]
 
+// Silence optional SDK pin warnings in CI
+env.SHARED_FEATURES_VERSION_OVERRIDE = ""
+// Avoid project-level parallelism to stabilize AGP data binding and transforms
+env.ORG_GRADLE_PARALLEL = "false"
+
 def pipeline = new org.android.AppPipeline(
     steps:            this,
     sonar:            true,
     sonarCmd:         'sonar -x :core-db:compileDebugUnitTestKotlin -x :core-db:compileDebugAndroidTestKotlin -x :feature-crowdloan-impl:compileDebugAndroidTestKotlin -x :runtime:compileDebugUnitTestKotlin -x :app:kaptDebugAndroidTestKotlin -x :app:compileDebugAndroidTestKotlin -Dsonar.coverage.jacoco.xmlReportPaths=**/coverage/*.xml',
     sonarProjectName: 'fearless-android',
     sonarProjectKey:  'fearless:fearless-android',
     pushReleaseNotes: false,
-    testCmd:          'runTest',
+    testCmd:          'postMergeVerify',
```

Comment:
I think this change should be discussed with the DevOps team. It looks like testCmd is intended to run tests (unit tests), while the new `postMergeVerify` does much more than just running tests.